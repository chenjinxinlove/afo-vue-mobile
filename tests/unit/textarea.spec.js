import { mount } from '@vue/test-utils'
import Demo from '../../src/textarea/demo'
import Vue from 'vue'
import Textarea from '../../src/textarea'

describe('Rate.vue', () => {
  // 测试是否可以被使用
  it('use', () => {
    Vue.use(Textarea)
    expect(Vue.component(Textarea.name)).toHaveProperty('name', 'VueComponent')
  })
  // demo测试快照
  it('demo snapshot', () => {
    expect(mount(Demo)).toMatchSnapshot()
  })
})