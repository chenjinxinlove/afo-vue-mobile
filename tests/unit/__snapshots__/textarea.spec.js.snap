// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rate.vue demo snapshot 1`] = `
<div>
  <p>基础用法</p>
  <div class="afo-textarea-wrapper">
    <textarea placeholder="please type here..." maxlength="60" indicator="true" class="afo-textarea"></textarea> <span class="afo-textarea__indicator" style="display: none;">50</span></div>
  <div style="height: 20px;"></div>
  <p>配置计数标识</p>
  <div class="afo-textarea-wrapper">
    <textarea placeholder="please type here..." maxlength="60" indicator="true" class="afo-textarea"></textarea> <span class="afo-textarea__indicator" style="display: none;">50</span></div>
  <p>多项配置</p>
  <div class="afo-textarea-wrapper">
    <textarea placeholder="请输入内容" maxlength="100" disabled="disabled" readonly="readonly" autofocus="autofocus" indicator="true" class="afo-textarea"></textarea> <span class="afo-textarea__indicator" style="display: none;">90</span></div>
</div>
`;
